### ISSUE TITLE
ERRORS during Building Apollo inside Container

### ISSUE THREAD

**Issue by snsmssss** [ApolloAuto/apollo#14713]
**Describe the bug**
After I have successfully finished setting up Apollo Dev Environment, 
During "Build Apollo inside Container" step 
when I run the following command

"bash docker/scripts/dev_into.sh"

I get the following ERRORs

1).

bazel-out/k8-opt/bin/modules/prediction/proto/prediction_conf.pb.h:143:3: note: declared here
   ObstacleConf_PredictorType_REGIONAL_PREDICTOR PROTOBUF_DEPRECATED_ENUM = 2,
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
(19:42:45) ERROR: /apollo/modules/prediction/BUILD:9:11: C++ compilation of rule '//modules/prediction:prediction_component_lib' failed (Exit 4): crosstool_wrapper_driver_is_not_gcc failed: error executing command external/local_config_cuda/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc -MD -MF bazel-out/k8-opt/bin/modules/prediction/_objs/prediction_component_lib/prediction_component.pic.d ... (remaining 185 argument(s) skipped)
In file included from ./modules/prediction/container/container_manager.h:30:0,
                 from ./modules/prediction/common/message_process.h:29,
                 from ./modules/prediction/prediction_component.h:29,
                 from modules/prediction/prediction_component.cc:17:

2).

x86_64-linux-gnu-gcc-7: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-7/README.Bugs> for instructions.
(19:42:46) INFO: Elapsed time: 1017.213s, Critical Path: 1004.37s
(19:42:46) INFO: 31 processes: 11 internal, 20 local.
(19:42:46) FAILED: Build did NOT complete successfully

I am unable to proceed to the next Step of

Launch and Run Apollo

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 -  NVIDIA-SMI 520.61.05    Driver Version: 520.61.05    CUDA Version: 11.8 
 - gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
 - Browser chrome,

**Smartphone (please complete the following information):**

**Additional context**

**Comment by daohu527** [comment]
do you have gpu and cuda installed?

If not, it can only be compiled in cpu mode. If have then use `nvidia-smi` to check if the gpu is already working.

**Comment by snsmssss** [comment]
Yes. I have 1 GPU and the near latest version of CUDA installed.
NVIDIA-SMI output
520.61.05 Driver Version: 520.61.05 CUDA Version: 11.8.

GPU/CUDA are all working fine

**Comment by daohu527** [comment]
@snsmssss Based on the information you provided, it is currently impossible to determine what the problem is. To be honest, I have never encountered a similar error. 

Maybe you need to analyze the error message first?
